File: Height-Checker--selection-sort.js
var heightChecker = function(heights) {
    let count=0
    let expected = heights.slice().sort((a,b)=>a-b)
    for (let i=0; i<heights.length;i++){
        if ( expected[i] !== heights[i]){
            count++
        }
    }
    return count
}   
console.log(heightChecker([10,6,6,10,10,9,8,8,3,3,8,2,1,5,1,9,5,2,7,4,7,7]))

File: Height-Checker--selection-sort.py
class Solution(object):
    def heightChecker(self, heights):
        """
        :type heights: List[int]
        :rtype: int
        """
        n=len(heights)
        count=0
        expected = sorted(heights)
        for i in range(n):
            if expected[i] != heights[i]:
                count+=1
        return count
        
n=input().strip("[]")
n=list(map(int,n.split(",")))
print(Solution().heightChecker(n))
